tee-worker/ts-tests-with-workspaces/sidechain-api/prepare-build/interfaces/definitions.ts

```diff
@@ -0,0 +1 @@
+export { default as dummynocustom } from "./dummynocustom/definitions";
```

Comment:
I had to add these; otherwise the type generation step would throw warnings and the package wouldn't compile

Reply:
PS if and when this all works, we should probably report it an issue to polkadotjs (not that I have high hopes that it'll be fixed ;P )

Reply:
>I had to add these; otherwise the type generation step would throw warnings and the package wouldn't compile

Moonbeam has also approached it [in the same way](https://github.com/PureStake/moonbeam/blob/19889514e6bd5f0f7e3b307e0e68b4c59df872ca/typescript-api/src/moonriver/interfaces/empty/definitions.ts), and I believe this could be an issue with the latest version of Typegen.